runtime/onert/backend/cpu/ops/SpaceToBatchNDLayer.cc

```diff
@@ -34,44 +34,11 @@ SpaceToBatchNDLayer::SpaceToBatchNDLayer()
   // DO NOTHING
 }
 
-// TO DO : move into shape inferer
```

Comment:
@hyunsik-yoon I removed this because I believe it will be done by shape inferer. However I wonder if shapeinferer runs always...

Reply:
Resolved by offline conversation : yes, shape inferer runs always for now. However further optimization is expected that not running StaticShapeInferer (neither Dynamic) when nnfw_set_input_tensorinfo is not called. So it's better remaining this.